src/D2L.CodeStyle.TestAnalyzers/Common/Diagnostics.cs

```diff
@@ -51,5 +51,15 @@ public static class Diagnostics {
 			isEnabledByDefault: true,
 			description: "Custom service locators are expensive to instantiate and slow down tests significantly. Use the default locator instead."
 		);
-	}
+
+        public static readonly DiagnosticDescriptor TestAttributeMissed = new DiagnosticDescriptor(
+            id: "D2LTESTS006",
+            title: "Method not labelled as [Test], [TestCase], or [TestCaseSource]",

```

Comment:
Ah, sorry, we also have some tests that only use `[Theory]`.

I've skimmed [NUnit's list of attributes](https://github.com/nunit/docs/wiki/Attributes), and I think this is the last "hey, this method is a test" attribute that we use.

Reply:
Added the theory attribute